package/MDAnalysis/coordinates/memory.py

```diff
@@ -514,7 +518,14 @@ def timeseries(self, asel=None, start=0, stop=-1, step=1, order='afc'):
 
         .. versionchanged:: 1.0.0
            Deprecated `format` keyword has been removed. Use `order` instead.
+        .. versionchanged:: 2.4.0
```

Comment:
use a `.. deprecated` and add more details on what's deprecated?

Reply:
I probably wouldn’t add a versionchanged here and only have the .. deprecated under the stop kwarg.